Fixed grammar, add GT 635, Kepler Gen1/2->600/700 (#9)
dortania/GPU-Buyers-Guide: modern-gpus/nvidia-gpu.md

Supported cards:

> 700-Series:

Reviewer: ```suggestion
700 Series:
```

* GTX Titan (GK 110 Maxwell core)
* GTX Titan Black(GK 110 Maxwell core)
* GTX Titan Z (One of the few dual GPU cards supported in macOS, but only one core will be in use)
* GTX 780 Ti
* GTX 780
* GTX 770
* GTX 760 Ti
* GTX 760
* GT 740
* GT 730 (GK208 variant)
* GT 720
* GT 710 (GK208 variant)

> 600-Series:

Reviewer: ```suggestion
600 Series:
```

#### Highest Supported OS: High Sierra(10.13.6)
#### Initial Supported OS: Lion(10.7)

> Since nVidia can't comply to any naming conventions, they decided to make some 600/700 series cards based on the older and not compatible Fermi architecture. These cards include:

Reviewer: ```suggestion
Since nVidia can't comply to any naming conventions, they decided to make some 600/700 series cards based on the older and incompatible Fermi architecture. These cards include:
```
